CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/atomist/automation-client-ts/compare/0.5.0...HEAD
 
+### Changed
```

Comment:
There should be an empty line between headers and their surrounding context. GitHub's markdown parser allows it, but others do not. It's not a big deal for this PR but might be something you want to ensure your automation does.